test/Mono.Linker.Tests.Cases/Warnings/WarningSuppression/SuppressWarningsInCompilerGeneratedCode.cs

```diff
@@ -344,55 +348,62 @@ class SuppressInLambda
 		{
 			// Suppression currently doesn't propagate to local functions
 
-			[ExpectedWarning ("IL2026", CompilerGeneratedCode = true)]
+			[ExpectedWarning ("IL2026", CompilerGeneratedCode = true, ProducedBy = ProducedBy.Trimmer)]
 			[UnconditionalSuppressMessage ("Test", "IL2026")]
 			static void TestCallRUCMethod ()
 			{
-				Action _ = () => RequiresUnreferencedCodeMethod ();
+				Action a = () => RequiresUnreferencedCodeMethod ();
+				a ();
```

Comment:
Why do we need this change?

Reply:
I used this in testing to see if there is a difference when the action is invoked (there wasn't) and forgot to remove it